lib/mixlib/install/options.rb

```diff
@@ -25,17 +25,7 @@ class Install
     class Options
       class InvalidOptions < ArgumentError; end
 
-      attr_reader :options, :errors, :original_platform_version
-
-      SUPPORTED_ARCHITECTURES = %w{
-        i386
-        powerpc
-        ppc64
-        ppc64le
-        s390x
-        sparc
-        x86_64
-      }
```

Comment:
I'm kind of on the fence with this... What this really represents is things that map to the `omnibus.architecture` in the metadata, which *should* be a fixed list, so I'm ok with leaving this in there since this is unlikely to change, however the mappings could change.

Does that make sense? I think I'm leaning towards leaving the array since it really shouldn't change. We should be mapping (by exception) to one of these in all cases.